src-tauri/src/main.rs

```diff
@@ -202,11 +202,17 @@ async fn kill_process(pid: u32, state: State<'_, AppState>) -> Result<bool, Stri
 
 fn main() {
     tauri::Builder::default()
+        .plugin(tauri_plugin_clipboard_manager::init())
+        .plugin(tauri_plugin_dialog::init())
+        .plugin(tauri_plugin_global_shortcut::Builder::new().build())
+        .plugin(tauri_plugin_fs::init())
+        .plugin(tauri_plugin_process::init())
+        .plugin(tauri_plugin_notification::init())
+        .plugin(tauri_plugin_http::init())
+        .plugin(tauri_plugin_shell::init())
+        .plugin(tauri_plugin_os::init())
```

Comment:
why adding this full list of plugins?

Reply:
In Tauri 1.0, many plugins were included by default potentially including unnecessary functionality. With Tauri 2.0, the framework has moved towards a more modular approach, where you explicitly add only the plugins you need.

Reply:
interesting, but still I'm not 100% sure they are required.. worth exploring a bit more

Reply:
I agree. The new version of Tauri brings a lot to the table

Reply:
> interesting, but still I'm not 100% sure they are required.. worth exploring a bit more

Not all of them are required, I'm exploring and testing right now.